### PR TITLE
More efficient default coder.

### PR DESCRIPTION
**Pull request by robertwb**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [ ] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [ ] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [ ] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

### CODE REVIEW

**Comment by robertwb** [comment]
R: @charlesccychen

File: sdks/python/apache_beam/coders/coders.py [hunk 1]
```diff
@@ -354,6 +354,49 @@ def value_coder(self):
...
+    if is_pair_like:
+      value['is_pair_like'] = True
+      value['component_encodings'] = [
+          self.as_cloud_object(is_pair_like=False),
+          self.as_cloud_object(is_pair_like=False)
+      ]
+
+    return value
+
+  # We allow .key_coder() and .value_coder() to be called on PickleCoder since
```
Comment (charlesccychen) [inline sdks/python/apache_beam/coders/coders.py]: "PickleCoder"

**Comment by charlesccychen** [comment]
Thanks!  This LGTM.

**Comment by dhalperi** [comment]
Please manually close the PR at your earliest convenience.

**Pull request closed without merging**